src/language-css/parser-postcss.js

```diff
@@ -151,10 +152,11 @@ function parseNestedCSS(node, options) {
 
       // Check on SCSS nested property
       if (isSCSSNestedPropertyNode(node, options)) {
-        node.isSCSSNesterProperty = true;
+        node.isSCSSNestedProperty = true;
       }
 
-      node.selector = parseSelector(selector);
+      // TODO: node.selector cannot be overwritten
+      node.selectorTemp = parseSelector(selector);
```

Comment:
This is still here because overriding `node.selector` raises an error, but [postcss-selector-parser](https://github.com/postcss/postcss-selector-parser) is still need until sass-parser exposes selector parsing.